packages/lib/server/service/InsightsRoutingBaseService.ts

```diff
@@ -414,27 +657,40 @@ export class InsightsRoutingBaseService {
     }
   }
 
-  private buildAttendeeSqlCondition(filterValue: TextFilterValue): Prisma.Sql | null {
+  private buildAttendeeSqlCondition(
+    filterValue: TextFilterValue,
+    attendeeColumn: "name" | "email" | "phone"
+  ): Prisma.Sql | null {
```

Comment:
We used to have one single attendee condition that checks both name and email. Now we have split it into explicit three filters (attendee name, attendee email, attendee phone number)